notify-debouncer-mini/src/lib.rs

```diff
@@ -342,12 +382,9 @@ pub fn new_debouncer_opt<F: DebounceEventHandler, T: Watcher>(
 /// Short function to create a new debounced watcher with the recommended debouncer.
 ///
 /// Timeout is the amount of time after which a debounced event is emitted or a continuous event is send, if there still are events incoming for the specific path.
-///
-/// If tick_rate is None, notify will select a tick rate that is less than the provided timeout.
 pub fn new_debouncer<F: DebounceEventHandler>(
     timeout: Duration,
```

Comment:
The docs for this method could say that you invoke the `new_debouncer_opt` with the default `Config`, thus you can point to it.